bin/cmd.js

```diff
@@ -19,8 +20,10 @@ import Yargs from 'yargs'
 import { hideBin } from 'yargs/helpers'
 import open from 'open'
 
-const { version: webTorrentCliVersion } = JSON.parse(fs.readFileSync(new URL('../package.json', import.meta.url)))
-const { version: webTorrentVersion } = JSON.parse(fs.readFileSync(new URL('../node_modules/webtorrent/package.json', import.meta.url)))
+const require = createRequire(import.meta.url)
+
+const { version: webTorrentCliVersion } = require('../package.json')
```

Comment:
This particular line did not previously cause any issue, but I changed this to use `require` for consistency.